src/lib.rs

```diff
@@ -0,0 +1,84 @@
+pub mod read_create {
+    extern crate calamine;
+    use calamine::{open_workbook, Xlsx};
+    use std::fs::File;
+    use std::io::BufReader;
+
+
+pub fn create_file(input_file:&str) -> File{
+
+    let file_handle = File::create(input_file).expect("unable to create a 01-03-2020.txt output file");
+     file_handle
+} 
+
+pub fn open_file(output_file:&str) ->  Xlsx<BufReader<File>>  {
+
+    let workbook: Xlsx<_> = open_workbook(output_file).expect("unable to open the TD_RateCard.xlsx excel file");
+    workbook
```

Comment:
don't need to return `workbook` variable u can  return `open_workbook(output_file).expect("unable to open the TD_RateCard.xlsx excel file")` directly

Reply:
i have tried it before it was showing error. Even now it is showing error.